### DISCUSSION TITLE
Monitor setup with multiple workstations

### DISCUSSION THREAD

**Discussion by twiclo** [hyprwm/Hyprland#5805] [Q&A]
I take my laptop to and from work. How should I go about setting up two different sets of monitor layouts? Should I I just have 6 different monitor lines then things will hopefully work because the ports match up at each location?

**Comment by twiclo** [comment]
The answer is [kanshi](https://sr.ht/~emersion/kanshi/)
Install
`sudo dnf install kanshi -y`
Setup your config in `~/.config/kanshi/config`
```
profile home {
        output eDP-1 disable
        output DP-11 position 0,0
        output DP-9 position 2560,0
}
profile mobile {
        output eDP-1 enable
}
```
Add an `exec-once` to hyprland

Something that I got hung up on was that each profile needs to match exactly what's plugged in. If every monitor that is picked up with `hyprctl monitors` isn't accounted for in a profile then that profile won't match

**Reply by Mr-Thack** [reply]
I think you dropped this:


![image](https://github.com/user-attachments/assets/db3038a8-adae-45d3-81e9-ef59336548e6)
